tableauserverclient/server/endpoint/datasources_endpoint.py

```diff
@@ -319,6 +319,71 @@ def update_connection(
         logger.info(f"Updated datasource item (ID: {datasource_item.id} & connection item {connection_item.id}")
         return connection
 
+    @api(version="3.26")
+    def update_connections(
+        self, datasource_item: DatasourceItem, connection_luids: list[str], authentication_type: str, username: Optional[str] = None, password: Optional[str] = None, embed_password: Optional[bool] = None
```

Comment:
Should accept an `Iterable[ConnectionItem]` rather than taking a list of ids and then Connection attributes as keyword args.